src/extensions/averagewaveforms2/AverageWaveformsView/waveformLayer.ts

```diff
@@ -0,0 +1,53 @@
+import { funcToTransform } from '../../CanvasWidget';
```

Comment:
Added a waveform layer. This layer goes on top of the electrodes layer, and shows the avg waveforms